### ISSUE TITLE
the output of I3D model and the calculation of loss function

### ISSUE THREAD

**Issue by punknownq** [tomrunia/PyTorchConv3D#13]
Hi,
Thank you for your work, firstly.
Do we need to add `x = x.mean(2)` in forward function of the I3D model?
And why do you add `        if config.model == 'i3d': 
            targets = torch.unsqueeze(targets, -1)` in train_epoch while it has not been added in validation_epoch?
